### ISSUE TITLE
ui framework integrations

### ISSUE THREAD

**Issue by theluk** [theluk/react-formular#1]
For better real-life usage, `react-formular` should be integratable into some popular ui frameworks like `Material` or `grommet`.

I was thinking about a sub package with some naming convention like `react-formular-{uiFramework}`.

**Comment by Pjaerr** [comment]
Are there specific libraries you want to integrate it with, or just create sub package for as many as possible?

**Comment by theluk** [comment]
Hey Pjaerr,

Thanks a lot for your question.

Regarding the fact that people are trying to decrease bundle size as much as possible, my first intention would be to make sub libs that only do the minimum required. 

But of course if webpack for example can take care of extracting modules from one package and then minimize the bundle, it would not be important to make a package for each UI library. 

So what do you think?

**Comment by Pjaerr** [comment]
Yes that makes sense, try and make it as modular as possible. I like the idea of this project so I will keep up to date and if I can contribute I definitely will :D
